api4/team.go

```diff
@@ -23,6 +23,7 @@ func InitTeam() {
 	BaseRoutes.Team.Handle("", ApiSessionRequired(getTeam)).Methods("GET")
 	BaseRoutes.Team.Handle("/stats", ApiSessionRequired(getTeamStats)).Methods("GET")
 	BaseRoutes.Team.Handle("/members", ApiSessionRequired(getTeamMembers)).Methods("GET")
+	BaseRoutes.Team.Handle("/members/ids", ApiSessionRequired(getTeamMembersByIds)).Methods("POST")
```

Comment:
Oh, can you change this and the one above it to use `BaseRoutes.TeamMembers`? Looks like we missed that before